narwhals/dataframe.py

```diff
@@ -1196,26 +1196,29 @@ def drop_nulls(self: Self, subset: str | list[str] | None = None) -> Self:
             The original object with the rows removed that contained the null values.
 
         Notes:
-            pandas and Polars handle null values differently. Polars distinguishes
-            between NaN and Null, whereas pandas doesn't.
+            pandas handles null values differently from Polars and PyArrow.
+            See https://narwhals-dev.github.io/narwhals/pandas_like_concepts/null_handling/
```

Comment:
Maybe we can use the links like: [null_handling](https://narwhals-dev.github.io/narwhals/pandas_like_concepts/null_handling) ?